src/components/http/Response.js

```diff
@@ -836,6 +838,14 @@ class Response {
         return this.#headers ? this.#headers[name] : undefined;
     }
 
+    /**
+     * ExpressJS: Returns all pending headers from this response 
+     * @returns {Object|undefined}
+     */
+    getHeaders() {
+        return this.#headers
```

Comment:
This method looks fine but just cross-check with the ExpressJS spec that the returned Object can support the following type structure: Object<string, string|Array<string>> because for multiple header values, the value of that header name key will be an array containing the multiple strings.

Reply:
Afaik uws is always going to return `{ [key: string]: string[] }` which is not exactly the same as express... but its notated in the types.  Express sends back an object `{ [key: string]: string }` where multiple values are probably concatenated.  If you want I can make it match express 1:1 but its going to require a little post processing of the #headers object.

Reply:
@kartikk221  Let me know what you want to do about this one.

Reply:
Try not to modify the structure of the original #headers object but rather simply return a generated fresh object with concatenated string values from the arrays, so this way all of the internals remain compatible and there will be slight post processing cost but its nothing major especially since compatibility is more important than a slight hit in performance in this case.

Reply:
Can do!